mls/nostr-mls-sqlite-storage/migrations/V104__add_image_hash_remove_image_url.sql

```diff
@@ -0,0 +1,7 @@
+-- groups: drop image_url, add image_hash (BLOB)
+ALTER TABLE groups DROP COLUMN image_url;
+ALTER TABLE groups ADD COLUMN image_hash BLOB;
```

Comment:
we should probably put quotes around groups as GROUPS is a reserved sqlite keyword. It doesn't seem to be causing issues now but I guess it would be a better practice.